source/index.html

```diff
@@ -1,30 +1,258 @@
+      </div>
+    </header>
+
+    <section id="about">
+      <div class="about-me">
+        <h2>Custom Website Design</h2>
+        <p>You’re here because you or your business need a website. Or maybe because you Google searched “pot pie” and “digital timer”? Which would be wierd. Well, the latter I cannot help with, but if it's the former, that is what I do! Take a look below to find out more about me. When you're done drop me a line, call or text and lets chat about your project.</p>
+      </div>
+      <div class="line-break">
+        <hr class="big-line">
+        <h3 class="title">This is Me</h3>
+        <hr class="small-line">
+      </div>
+      <div class="my-info">
+        <img class="display766" src="./assets/index-pano.png">
+        <img src="./assets/family.png">
+        <div class="family-man">
+          <p>Personally, I am a husband, and father of two little inspirations. I am a Yooper: <em> a native or inhabitant of the Upper Peninsula of Michigan.</em> I am an avid photographer, beer brewer, wood worker and at a time in my life, long distance runner. I love all things outdoors and picked up web development as a creative outlet, but ended up turning it into a lifestyle and career.</p>
+          <p>Proffesionally, I am a strategic and innovative thinker with 13 years of business development and management experience and 3 years of self-education in web design and development. I’ve completed a handful of projects in the past year. My strengths include creativite problem solving, tenacity, and resourcefulness. I love working closely with people on their passion projects to help create and build custom solutions that will bring success to their business, organization or whatever! So if you are looking for a custom solution to a web based problem. <a href="mailto:mark.m.hayes1@gmail.com">Contact me</a> and let's chat about how I can help.</p>
+        </div>
+      </div>
+    </section>
+    <section class="kind-words">
+      <h2>Testimonials</h2>
+      <div class="why-ppd">
+      <div class="testimonial">
+        <img src="./assets/testimonial_martha.png" alt="Martha Parks">
+        <p>Pot Pie Digital is one of the hardest working and reliable companies I know. They were determined and thorough and would not quit until they designed an awesome website that would work smoothly for Panara Imports. They listened to our needs and more than met our expectations.</p>
+        <h5>Martha Parks</h5>
+        <h5>Owner and Operator Panara Imports</h5>
+      </div>
+      <div class="testimonial">
+        <img src="./assets/testimonial_shana.png" alt="Shana Paveglio Kleinschmidt">
+        <p>Pot Pie Digital was exceptional to work with, they listened to my ideas and brought many fun ideas of their own to help build our cite. My web page turned out great and is easy for all of my customer to navigate. Would definitely recommend them for anyone looking to build a web page for the business.</p>
```

Comment:
`site` (not `cite`)

Reply:
`web-page` (needs hyphen)

Reply:
`customers` (plural)

Reply:
`I would definitely recommend them for anyone looking to build a web-page for their business.` (`I`, `web-page`, and `their` … just phrasing)